plugins/tiddlywiki/menubar/plugin.info

```diff
@@ -2,5 +2,7 @@
 	"title": "$:/plugins/tiddlywiki/menubar",
 	"name": "Menu Bar",
 	"description": "Menu Bar",
-	"list": "readme config"
+	"list": "readme config license",
+	"plugin-priority": "99999",
```

Comment:
ATM plugins with an existing translation will need a `plugin-priority`, otherwise the add-on will not be activated. @Jermolene it would be nice to avoid this new parameter, but I couldn't find a good way.